WooCommerce/Classes/ViewRelated/Orders/Order Creation/EditableOrderViewModel.swift

```diff
@@ -1025,6 +1025,9 @@ final class EditableOrderViewModel: ObservableObject {
             guard let self else { return }
             self.onFinished(order)
             self.trackCreateOrderSuccess(usesGiftCard: usesGiftCard)
+            Task {
+                await PointOfSaleNotificationScheduler().scheduleLocalNotificationIfEligible(for: .potentialMerchant)
+            }
```

Comment:
Task itself could hold it while it's being executed.
```suggestion
            let scheduler = PointOfSaleNotificationScheduler()
            Task { [scheduler] in
                await scheduler.scheduleLocalNotificationIfEligible(for: .potentialMerchant)
            }
```